src/main/java/br/com/deveficiente/bolaoapi/shared/api/ExceptionController.java

```diff
@@ -0,0 +1,30 @@
+package br.com.deveficiente.bolaoapi.shared.api;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@RestControllerAdvice(annotations = RestController.class)
```

Comment:
*Usamos tudo que conhecemos que está pronto. Só fazemos código do zero se for estritamente necessário.*

O RestControllerAdvice já pega so RestController :).

Reply:
Hmm não sabia!
Obrigado pela dica!!!